### DISCUSSION TITLE
Using Symbol Mapping - which data enhancers are allowed

### DISCUSSION THREAD

**Discussion by morskipas** [ghostfolio/ghostfolio#3873] [Q&A] [answered]
Hello everyone,
I have the following question. I would like to show the allocations of the DWS Vermögensbildungsfonds I (DE0008476524) in Ghostfolio.
For some of my ETFs, I could do this using symbol mapping, for example {“TRACKINSIGHT”: “IWDA”} 
Unfortunately, Trackinsight only recognizes ETFs and no funds. What other values are possible here instead of Trackinsight to make a mapping?

Thank you for your support
Kind regards
Frank

**Comment by dtslvr** [comment]
Hi @morskipas

`TRACKINSIGHT` is currently the only one. If you don't find your fund there, you could add countries / sectors manually in the _Admin Control_ panel.

**Reply by morskipas** [reply]
Hi @dtslvr,

how and where can I do this?
![image](https://github.com/user-attachments/assets/0d5f2fbb-72b2-4b86-878b-a4cd28a1dc2b)

**Reply by dtslvr** [reply, marked as answer]
My bad, it is only possible to do it directly in the GUI if the data source is `MANUAL`.
You could still add countries / sectors in the database table `SymbolProfileOverrides`.

**Reply by LapplandsCohan** [reply]
> My bad, it is only possible to do it directly in the GUI if the data source is `MANUAL`. You could still add countries / sectors in the database table `SymbolProfileOverrides`.

Are there any documentation or hints/instructions of how to do this?
